### PR TITLE
Patch

### PR DESCRIPTION
**Pull request by ayushseth07**
### Description
To Do List issue no #196 

### Checklist

- [x] I've been assigned an issue related to this PR.
- [x] I've used beautifiers.
- [x] I've added my Project's name and description to `Index.md`
- [x] I've made a README.md file for my Project.
- [x] The README.md file of my project contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number

(Write your answer here.)

### CODE REVIEW

File: todolist/Readme.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```
+* Open http://localhost:3000/ on your browser.
+<br/>
+
+<h3>ScreenShots</h3>
+<br>
+<img src="images/main.PNG"/>
+<img src="images/work.PNG"/>
+
+
```
Comment (chicken-biryani) [inline todolist/Readme.md:49]: Remove the extra line at the end

File: todolist/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="utf-8">
+    <title>To Do List</title>
+  </head>
+  <body>
+    <h1>It's a weekday!</h1>
+    <p>Why are you not working!</p>
+
+
+  </body>
+</html>
```
Comment (chicken-biryani) [inline todolist/index.html:13]: Use an html beautifier. Add a new line at the eof.

File: todolist/views/list.ejs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      </div>
+      <% } %>
+
+      <form class="item" action="/" method="post">
+        <input type="text" name="newItem" placeholder="New Item" autocomplete="off">
+        <button type="submit" name="list">+</button>
+      </form>
+  </div>
+
+<%- include("footer") -%>
```
Comment (chicken-biryani) [inline todolist/views/list.ejs:21]: Use a js beautifier.

add a new line at the eof.

File: todolist/views/header.ejs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>To Do List</title>
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>
```
Comment (chicken-biryani) [inline todolist/views/header.ejs:10]: Use an HTML beautifier....

File: todolist/Readme.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+node app.js
+```
+* Open http://localhost:3000/ on your browser.
+<br/>
+
+<h3>ScreenShots</h3>
+<br>
+<img src="images/main.PNG"/>
+<img src="images/work.PNG"/>
+
```
Comment (chicken-biryani) [inline todolist/Readme.md:48]: The screenshots aren't working.

File: todolist/views/about.ejs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<%- include("header") -%>
+
+<h2>This is the about page</h2>
+
+<p>jkhkjshjks dhfjkh sjkdfh sjkdf jksdfhjksfhjks dhjfk sjkdfjksdfhjkshdfjk sdjkf sjkdf jksd fjks djfkhsjkdf hsjkd fhs</p>
```
Comment (harshita2216) [inline todolist/views/about.ejs:5]: What is this? 🤣

**Comment by harshita2216** [comment]
@ayushseth07, just add the project details to index.md and then it would be good to go! 🎉

**Comment by ayushseth07** [comment]
> @ayushseth07, just add the project details to index.md and then it would be good to go! 🎉

@harshita2216 i have added the details in readme.md

**Comment by harshita2216** [comment]
> > @ayushseth07, just add the project details to index.md and then it would be good to go! tada
> 
> @harshita2216 i have added the details in readme.md

https://github.com/Ayushparikh-code/Web-dev-mini-projects/blob/main/Index.md

Hey, I'm talking about this file. Add your details in a manner given in this file.

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**